assignments/objects.js

```diff
@@ -19,46 +19,116 @@ const example = {
 
 // Write your intern objects here:
 
+const Mitzi = {
+  id: 1,
+  name: 'Mitzi',
+  email: 'mmelloy@psu.edu',
+  gender: 'F'
+}
+
+const Kennan = {
+  id: 2,
+  name: 'Kennan',
+  email: 'kdiben1@tinypic.com',
+  gender: 'M',
+  speak: function speak() {
+    console.log(`Hello, my name is Kennan!`)
+  }
+}
+
+const Keven = {
+  id: 3,
+  name: 'Keven',
+  email: 'kmummery2@wikimedia.org',
+  gender: 'M'
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
+const Gannie = {
+  id: 4,
+  name: 'Gannie',
+  email: 'gmartinson3@illinois.edu',
+  gender: 'M'
+}
+
+const Antonietta = {
+  id: 5,
+  name: 'Antonietta',
+  email: 'adaine5@samsung.com',
+  gender: 'F',
+  multiplyNums: function multiplyNums(x, y) {
+    console.log(x*y);
+  }
+}
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(Mitzi['name']);
```

Comment:
This is totally fine. I'm just suggesting dot notation because it might be a little quicker and cleaner.

```suggestion
console.log(Mitzi.name);
```